package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "parse-server",
-  "version": "2.6.5",
+  "name": "@parse/server",
+  "version": "3.0.0-alpha.0",
   "description": "An express module providing a Parse-compatible API server",
```

Comment:
Should we change the description while we're at it? Since this _is_ the server now. Could word it like
> An express module for the Parse backend server
Just a thought.

Reply:
Yeah, still can be used as an express app as it’s an express app that can be started.